### PR TITLE
feat(seed) successfully seeds some dummy data

### PR DESCRIPTION
**Pull request by danielhoward314**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

**Review by kkomarocker — APPROVED** [review]

File: script/seed.js [hunk 1]
```diff
@@ -39,7 +93,7 @@ if (module === require.main) {
     console.error(err)
     process.exitCode = 1
   })
-  .finally(() => { // `finally` is like then + catch. It runs no matter what.
+  .then(() => { // `finally` is like then + catch. It runs no matter what.
```
Comment (glebec) [inline script/seed.js:96]: Why did you need to do this? `finally` works on my machine!

_this is an example code review comment._
